gcloud/streaming/http_wrapper.py

```diff
@@ -64,14 +65,14 @@ def _Httplib2Debuglevel(http_request, level, http=None):
     an httplib2.Http object is provided, we'll also change the level on
     any cached connections attached to it.
 
-    Args:
```

Comment:
Does `tox -e docs` fail on the Google-style docstrings? Have you done any sort of systematic check for them?